### DISCUSSION TITLE
Option to bring up classic context menu with Alt + Right click

### DISCUSSION THREAD

**Discussion by Z-Dante** [valinet/ExplorerPatcher#269] [Ideas]
Hi there. 

First of all, I'd like to start this post by saying that I like both the old and the new context menus and feel like both has their uses. The new menu is great at basic tasks while the old menu is great at productivity. So the ability to easily access both with the right-click and minimal hand movement would be great for productivity. 
I know it is possible to always get the old context menu with right-click but we lose the new context menu that way. And the current shortcut key to bring up the context menu (Shift+F10) requires both hand movements and does not place the context menu right next to the mouse pointer either. So a simple hold alt button and right-click to bring up the classic context menu and normal right-click to bring up the new context menu (or vice versa) would be a lifesaver for me. 

As of Windows 11 build 22494, the alt + right-click key combination does absolutely nothing so we wouldn't lose anything this way. So if it's possible I'd like to request the feature to bring up the classic context menu with the alt + right-click button while the new context menu is not disabled. Or the ability to bring the new context menu if the old menu is enabled.

**Comment by valinet** [comment]
Hi

Thanks for the suggestion. It is pretty complicated to implement. But like, anything’s possible, no doubt about that.

As a general answer to most of the requests, not targeted there specifically, please keep in mind that I am not Microsoft, Explorer is not my software, everything this patcher implements took a lot longer to research and implement as the vast majority of knowledge comes from disassembly; I don’t have any access to the actual source code, and without, everything is very tedious to do.

I mean, I understand the vast majority of the requests I receive, and they make sense. The problem is, it’s pretty hard to fix each and every flaw in this OS. I am just a single person, with limited time. I don’t have time to work forever on this project… also, Microsoft should also take the time and fix its OS…

This being said, idk, if I figure out a way to do this relatively quickly, it’s there, I noted it, I will think about it. Also, if you have technical ideas how to implement this, I am all ears, contributions are always welcome.

Thanks
